markdown.dtx

```diff
@@ -18934,7 +19167,93 @@ end
 %    \end{macrocode}
 % \par
 % \begin{markdown}
-%
+% The most comfortable way to to parse the jekyllData is using \pkg{expl3}'s
+% l3keys module. Due to the structure of the data we need to initialize some
+% variables to save the data.
+% \end{markdown}
+%  \begin{macrocode}
+\ExplSyntaxOn
```

Comment:
How portable is this? Currently, we support TeX Live ≥ 2013. Is it sufficient to add `\RequirePackage{expl3}` to keep support for TeX Live ≥ 2013?

Reply:
Well there has been huge progress on expl3 during the last years.  I did not expect anyone would use 2013 anymore … 

It's easy to implement it on … let's estimate 2017+ but I'd not support more than required, because one would have to add fallbacks for some renaming things …  
I usually expect users to be able to update their package to also update their distribution …

Reply:
Good to know. If we need to stop supporting legacy versions of TeX Live, then we will, but it's good to have this documented. What I am thinking is this: I will add matrix testing for historical versions of TeX Live to the GitHub Actions workflow in `main` and we will see what breaks after we have added the expl3 implementation. Then we will see how far back we can go. 🙂